tests/extensions/provisioning/verify.go

```diff
@@ -56,13 +56,19 @@ func VerifyCluster(t *testing.T, client *rancher.Client, clusterName string, ter
 	clusterID, err := clusters.GetClusterIDByName(adminClient, clusterName)
 	require.NoError(t, err)
 
-	waitState.IsActiveCluster(adminClient, clusterID)
-	waitState.AreNodesActive(adminClient, clusterID)
+	if err := waitState.IsActiveCluster(adminClient, clusterID); err != nil {
+		require.NoError(t, err)
+	}
+
+	if err := waitState.AreNodesActive(adminClient, clusterID); err != nil {
```

Comment:
nit; this is different than our normal format of having it broken out on separate lines

Reply:
Yeah. There is a very specific error where after a cluster is finished provisioning/upgrading where it does not proceed to the next step in this file. This was acting a safeguard against that issue.